software/README.md

```diff
@@ -28,3 +28,37 @@ $ pip-compile software/requirements_dev.in
 ```
 
 Both the ``software/requirements_dev.in`` and the generated ``software/requirements_dev.txt`` files should be committed.
+
+## Configuration Customization
```

Comment:
I'm open to moving it to a different file. Editing the on-pi config file is sort of borderline between development & runtime options, since it has the ability to change the values of some constants (e.g. `MAX_OUTPUT_VOLTAGE` if you're working with a non-Eurorack system where 10V outputs could be actively dangerous)

I'll leave the docs where they are for the moment, but moving them will be pretty trivial if we decide to move forward and merge this.

Reply:
I gotta agree with @mjaskula, I'd move it into one called `configuration.md`, this way we can link to it in any other README file or if someone asks about it in Discord etc

Reply:
Good points, both of you. I see how this is a little dev-time and a little run-time, and I can definitely see us linking to this doc directly. I reiterate my vote for a separate file.

Reply:
[required] pull this documentation into a separate file.

Reply:
Done.